### PR TITLE
Issue 78 add decorator for modal portals to bind to empty div

### PR DESCRIPTION
**Pull request by HacksForHugs**
- Added a decorator in storybook config that adds a div with id="modal-root" for portal modals to bind to

### CODE REVIEW

**Comment by ductiletoaster** [comment]
We will need to publish a new version. Let me know once you get this in and I can get that going

File: packages/pixeloven-docs/src/configs/config.tsx [hunk 1]
```diff
@@ -30,6 +31,20 @@ addDecorator(
 );
 addDecorator(withKnobs);
 
+
+/**
+ * Wrapper with custom id for modals to attach to
+ */
+const modalContainerDecorator = (storyFn: () => {}) => (
+    <>
+        <div id= "modal-root" />
```
Comment (jimmynotjim) [inline packages/pixeloven-docs/src/configs/config.tsx]: Is the space between `=` and `"` a concern?

File: packages/pixeloven-docs/src/configs/config.tsx [hunk 2]
```diff
@@ -30,6 +31,20 @@ addDecorator(
 );
 addDecorator(withKnobs);
 
+
+/**
+ * Wrapper with custom id for modals to attach to
+ */
+const modalContainerDecorator = (storyFn: () => {}) => (
+    <>
```
Comment (jimmynotjim) [inline packages/pixeloven-docs/src/configs/config.tsx]: Never seen empty tags before, is this like `React.Fragment`?

File: packages/pixeloven-docs/src/configs/config.tsx (hunk 1 above)
Comment (jimmynotjim) [reply packages/pixeloven-docs/src/configs/config.tsx]: Should the modal root follow the normal root so it's more like what we'd expect to see in our own project?

Comment (HacksForHugs) [reply packages/pixeloven-docs/src/configs/config.tsx]: I have the modal div on top in our project

File: packages/pixeloven-docs/src/configs/config.tsx (hunk 2 above)
Comment (HacksForHugs) [reply packages/pixeloven-docs/src/configs/config.tsx]: Yeah!! It’s react fragment! I have some of this in our project too, it’s very shorthand. Tsx supports :)

File: packages/pixeloven-docs/src/configs/config.tsx (hunk 1 above)
Comment (HacksForHugs) [reply packages/pixeloven-docs/src/configs/config.tsx]: I can remove the space

Comment (jimmynotjim) [reply packages/pixeloven-docs/src/configs/config.tsx]: Ah ok, I had misunderstood the demo and thought the modal root was at the end of the `body`.

**Comment by ductiletoaster** [comment]
Not doing anymore. Went a different direction

**Pull request closed without merging**